### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by JannikWibker**
The code style got pretty out of control as of recently and our current linting setup isn't detecting it. Thus this PR adds prettier which is much stricter in enforcing and fixing coding style issues.

### CODE REVIEW

File: tasks/pages/forgot-password.tsx [hunk 1]
```diff
@@ -1,11 +1,7 @@
 import type { NextPage } from 'next'
 
 const ForgotPasswordPage: NextPage = () => {
-  return (
-    <div>
-      not sure how this is going to look or even work, just a placeholder right now
-    </div>
-  )
+  return <div>not sure how this is going to look or even work, just a placeholder right now</div>
```
Comment (MaxSchaefer) [inline tasks/pages/forgot-password.tsx:4]: We should at least stick with the leading and trailing brackets.

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
My two cents; We should keep:
- Commas overall (especially for `type`)
- Trailing commas, to prevent unnecessary git additions

**Comment by JannikWibker** [comment]
@MaxSchaefer I kind of agree with both your comments, but there is one problem with this:
- prettier is pretty opinionated and doesn't really allow for rules to be enforced in a non-strict way; meaning if you "allow" trailing commas you need them everywhere, there is no in between and taking that into account i'm more on the "against"-side when it comes to trailing commas
- prettier is light years ahead of other code formatting and linting tools, I used to dislike prettier because of the highly opinionated and non-configurable nature but have grown to rely on it really much as no other tool detects and fixes this many issues; thus not using prettier is in my opinion not really an option

**Comment by MaxSchaefer** [comment]
> @MaxSchaefer I kind of agree with both your comments, but there is one problem with this:
> 
> * prettier is pretty opinionated and doesn't really allow for rules to be enforced in a non-strict way; meaning if you "allow" trailing commas you need them everywhere, there is no in between and taking that into account i'm more on the "against"-side when it comes to trailing commas
> * prettier is light years ahead of other code formatting and linting tools, I used to dislike prettier because of the highly opinionated and non-configurable nature but have grown to rely on it really much as no other tool detects and fixes this many issues; thus not using prettier is in my opinion not really an option

We should not adopt code styles because the tools cannot be configured to our liking. I've skimmed over the changes, with a stricter configuration, ESLint should be able to handle this as well.

**Comment by use-to** [comment]
Houston, we have a problem!

**Pull request closed without merging**